src/pages/search.js

```diff
@@ -21,3 +21,13 @@ export default function Search() {
         </>
     );
 }
+
+Search.getInitialProps = async ({ Component, ctx }) => {
+    let pageProps = {};
+
+    if (Component.getInitialProps) {
+        pageProps = await Component.getInitialProps(ctx);
+    }
+
+    return { pageProps, namespacesRequired: ["common"] };
```

Comment:
The other pages have a shortened version of the `getInitialProps` function - is there a reason this one is different?

Reply:
Nope..I just forgot to update this.